projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -1,41 +1,66 @@
-import { Component, ChangeDetectionStrategy } from '@angular/core';
-// import { AbstractProductComponent } from '../abstract-product-component';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ViewChild,
+  ElementRef,
+} from '@angular/core';
+import { CMSTabParagraphContainer } from '@spartacus/core';
+import { CmsComponentData } from 'projects/storefrontlib/src/cms-structure';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
 
 @Component({
   selector: 'cx-tab-paragraph-container',
   templateUrl: './tab-paragraph-container.component.html',
-  styleUrls: ['./tab-paragraph-container.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TabParagraphContainerComponent {
-  // extends AbstractProductComponent {
-  // @ViewChild('tabContent') tabContent: Element;
-  // fetchData() {
-  //     // console.log(this.contextParameters.tab);
-  //     // this.contextParameters.tab.title = new BehaviorSubject<any>({});
-  //     // this.contextParameters.tab.title.next('test');
-  //     // this.contextParameters.tab.title.complete();
-  //     // const s = new AsyncSubject<any>();
-  //     // this.occProductSearchService.query(query)
-  //     //     .then((pageData) => {
-  //     //         s.next(pageData);
-  //     //         s.complete();
-  //     // });
-  //     // return s;
-  //     // if (!this.contextParameters.tab) {
-  //     //     this.contextParameters.tab = {};
-  //     // }
-  //     // this.contextParameters.tab.title = this.component.title;
-  //     // console.log(this.contextParameters.tab);
-  //     // this.cd.detectChanges();
-  //     // console.log(this.component.title);
-  //     // console.log(this.tabContent);
-  //     // console.log(this.contextParameters);
-  //     // this.contextParameters);
-  // }
-  // ngOnInit() {
-  //     this.contextParameters.tabTitle.next('test');
-  //     this.cd.detectChanges();
-  //     // super.fetchData();
-  // }
+  activatedElements: HTMLElement[] = [];
+
+  @ViewChild('header')
+  set initial(ref: ElementRef) {
+    if (ref) {
+      ref.nativeElement.click();
+    }
+  }
+
+  constructor(
+    public componentData: CmsComponentData<CMSTabParagraphContainer>
+  ) {}
+
+  components$: Observable<any> = this.componentData.data$.pipe(
+    map((data: CMSTabParagraphContainer) =>
```

Comment:
What about using the OCC component list API here? The GET version of that is about to be ready, see #1033.

Reply:
Not too sure which part you are referring to?